src/main/java/frc/robot/utils/ColorSensor.java

```diff
@@ -44,8 +44,7 @@ public ColorValue getColor() {
         return ColorValue.getFromColor(match.color);
     }
 
-    public ColorMatchResult getRawColor() {
-        Color detectedColor = colorSensor.getColor();
-        return colorMatcher.matchClosestColor(detectedColor);
+    public Color getRawColor() {
```

Comment:
Before we merge this, did you verify this is only used for logging purposes?

Reply:
I just made another method that does what the old one did since I forgot that probability isn't part of color

Reply:
What's the difference between "getColor()" and "getMatchedColor"? Instead of creating different public methods can you add one public method that exposes what people should be using?

Reply:
As far as I can tell getColor gives a ColorValue that's used to get the enum with the name in it whilst getMatched Color gives a ColorMatchResult that's used in the feeder to get the probability of the sensor being right.